src/components/tooltip/bl-tooltip.stories.mdx

```diff
@@ -0,0 +1,141 @@
+import { html } from 'lit';
+import { ifDefined } from 'lit/directives/if-defined.js';
+import { styleMap } from 'lit/directives/style-map.js';
+import { unsafeHTML } from 'lit/directives/unsafe-html.js';
+import { Meta, Canvas, ArgsTable, Story, Preview, Source } from '@storybook/addon-docs';
+
+<Meta
+  title="Components/Tooltip"
+  component="bl-tooltip"
+  argTypes={{
+    placement: {
+        control: 'text'
+    }
+  }}
+/>
+
+export const IconTriggerTemplate = (args) => html`
+<bl-tooltip placement="${ifDefined(args.placement)}" style='--bl-tooltip-position:fixed'>
+  <bl-icon slot="tooltip-trigger" name="settings"></bl-icon>
```

Comment:
```suggestion
  <bl-button icon="settings" text label="Settings"></bl-button>
```
We don't want to promote using `bl-icon` everywhere by itself. Instead -most of the time- people should use icon-only text buttons. So let's give good examples here.